src/main/java/de/hpi/datastreams/apps/App.java

```diff
@@ -0,0 +1,56 @@
+package de.hpi.datastreams.apps;
+
+import de.hpi.datastreams.processors.ServerProcessor;
+import de.hpi.datastreams.processors.WorkerSamplingProcessor;
+import de.hpi.datastreams.processors.WorkerTrainingProcessor;
+import de.hpi.datastreams.serialization.JSONSerde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.Topology;
+import org.apache.kafka.streams.state.Stores;
+
+import java.util.Properties;
+
+public class App extends BaseKafkaApp {
+
+    public final static String INPUT_DATA_TOPIC = "INPUT_DATA";
+    public final static String INPUT_DATA_BUFFER = "INPUT_DATA_BUFFER";
+
+    public final static String GRADIENTS_TOPIC = "GRADIENTS_TOPIC";
+    public final static String WEIGHTS_TOPIC = "WEIGHTS_TOPIC";
+
+    public static Integer START_VC = -1;
```

Comment:
Please add a comment explaining why it starts with `-1`.

Reply:
done